cypress/integration/anton.sample/todoSample.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Managing of todo list
+  As a user
+  I want to use todo list functionality
+  So I can plan my activities
+
+  Background:
+    When I add items via localStorage
+    When I open todo list page
+    Then I see that header is shown
+    Then I see that "item1" item is presented
+    Then I see that footer is shown
+
+  Scenario: Add new item into the list
+    When I type "Test1" into input field and press Enter
```

Comment:
Avoid using imperative type of writing. Use [declarative](https://wiki.saucelabs.com/display/DOCS/Best+Practice%3A+Imperative+v.+Declarative+Testing+Scenarios): `I add new "Test1" item`. That's it.